docs/crypto.md

```diff
@@ -21,6 +21,17 @@ sha256sum "Hello world!"
 The above will compute the SHA 256 sum in an "ASCII armored" format that is
 safe to print.
 
+## sha512sum
+
+The `sha512sum` function receives a string, and computes it's SHA512 digest.
```

Comment:
Wrong "its" :eyes:

```suggestion
The `sha512sum` function receives a string, and computes its SHA512 digest.
```

(I guess this was copied from `sha256sum` which has the same typo, but notably `adler32sum` and `bcrypt` below _don't_ have this typo :sweat_smile:)